extensions/wikia/Recirculation/js/views/rail.js

```diff
@@ -0,0 +1,46 @@
+/*global define*/
+define('ext.wikia.recirculation.views.rail', [
+	'jquery',
+	'wikia.window',
+	'wikia.log',
+	'wikia.mustache',
+	'ext.wikia.recirculation.tracker',
+	'ext.wikia.recirculation.utils'
+], function ($, w, log, Mustache, tracker, utils) {
+
+	var logGroup = 'ext.wikia.recirculation.views.rail';
+
+	function render(data) {
+		var deferred = $.Deferred();
+
+		utils.loadTemplate('extensions/wikia/Recirculation/templates/client/rail.mustache')
```

Comment:
The whole utils.loadTemplate pattern seems repeated in the view. Is there a simple way to avoid this duplication?